docs/user-documentation/objects.md

```diff
@@ -93,3 +93,21 @@ All nodes can contain any number of media.  The media for any node can be manage
 the "Members" tab, Actions can be performed in bulk using the check boxes and Actions drop down.
 
 ![Media tab](../assets/islandora_8_derivatives_sample.png)
+
+## Display Hints
+
+Media are displayed in Islandora 8 via "view modes". View modes are Drupal's name for alternative ways to present content to users. In Islandora, the view mode determines whether a media is displayed using the OpenSeadragon viewer or the PDFjs viewer, for example. View modes can be set explicitly in the node edit form, or you can configure Islandora to use a specific view mode for all media with a specific Mime type.
```

Comment:
I think this paragraph is slightly misleading. The view modes are set for the nodes, not the media. It just seems like it applies to the media because the primary difference between the node's view modes are which entity view attachments (which display media as a field) they include. Node view display modes also include default, binary, and teaser.

Reply:
Thanks, I'll clarify that.

Reply:
OK, let me know if there's anything else on this one.